### ISSUE TITLE
Demonstrate a database-centric ETL-like batch processing app -- as distinct from a message/REST focused application

### ISSUE THREAD

**Issue by GalateaEric** [GalateaRaj/fuse-starter-java#232]
Issue Description
--
Demonstrate a database-centric ETL-like batch processing app -- as distinct from a message/REST focused application

Design
--
Talk to Andy for some details  see comment on issue: #53 

Documentation Changes
--
[Description of changes necessary to the Fuse documentation due to the changes in response to this issue. Should be complete before Review is approved]

Test Evidence
--
[include relevant tests and output. Should be complete before Review is approved]

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]

**Comment by mikegajda** [comment]
An ETL-like application would be a different use case from the current REST API Fuse implementation. Thinking is that an ETL-like app should live in its own repo, maybe fuse-starter-java-etl? For now, leave this out-of-scope for this repo

**Comment by mikegajda** [comment]
Comment from Eric is that this would still be useful to demonstrate. Likely not in scope for fuse-starter-java, a good candidate for its own project, if time/resources permit.

**Comment by cbaltera** [comment]
See https://github.com/galatea-associates/fuse-starter-etl
